src/pages/result-page/ResultPage.tsx

```diff
@@ -24,12 +41,16 @@ export default function ResultPage(props: Props) {
   const classes = useStyles(props);
   const { tab } = useParams<Params>();
   const { id: revieweeId } = useAuthGuardUser();
-
+  const data = useLazyLoadQuery<ResultPageQuery>(query, { id: revieweeId });
   const toPrefix = '';
 
   return (
     <Container maxWidth="md" className={classes.container}>
       <Box marginY={5}>
+        <QuoteBox border="1px solid #00800033" textAlign="center" color="green" mb={2} bgcolor="success.light">
+          <Typography variant="h6">{`${i18n._('Overall Evaluation')}: `}</Typography>{' '}
+          <EvaluationOutput type="peer" value={data.viewer.user?.managerPersonReview?.overallRating as Evaluation} />
```

Comment:
why you cast this ? 
is all nullish operators required ?

Reply:
yes this is graphql type